lib/api/v3/utilities/custom_field_injector.rb

```diff
@@ -361,9 +347,10 @@ def resource_type(custom_field)
         end
 
         def allowed_users_static_filters
-          [{ status: { operator: '!',
+          [
+            { status: { operator: '!',
                        values: [Principal.statuses[:locked].to_s] } },
-           { type: { operator: '=', values: ['User'] } }]
```

Comment:
I'd rather keep this filter in here and have it extended with the values for Group and PlaceholderUser than to remove it. For reasons lost in time it is currently not possible to filter by user cf globally but that restriction could be removed rather sooner than later. And if it is, values like 'SystemUser' might creep in again without us noticing.

Reply:
:+1: